src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -839,9 +839,9 @@ class DetailsCard extends React.Component {
               </Row>
             </Grid>
             {/* Advanced options */}
-            { isFullEdit && <ExpandCollapseSection id={`${idPrefix}-advanced-options`} sectionHeader={msg.advancedOptions()}>
+            { isFullEdit && <ExpandCollapse id={`${idPrefix}-advanced-options`} className={style['adv-options-divider']} textCollapsed={msg.advancedOptions()} textExpanded={msg.advancedOptions()}>
```

Comment:
Can the CSS manipulation to make the PF3 `ExpandCollapse` component look more like our `ExpandCollapseSection` be done by modifying the PF3 css?

Those mods can go in `index-nomodules.css`.  That would be preferred so the `adv-options-divider` doesn't need to be specified on a component each time the `ExpandCollapse` is used.

Relevant classes:
  - `expand-collapse-pf`
  - `expand-collapse-pf-link-container`
  - `expand-collapse-pf-separator`

Reply:
You can see them in use in the component over at: https://github.com/patternfly/patternfly-react/blob/patternfly-react%402.39.16/packages/patternfly-react/src/components/ExpandCollapse/ExpandCollapse.js

Reply:
https://github.com/oVirt/ovirt-web-ui/issues/1264#issuecomment-696258357